versions.txt

```diff
@@ -0,0 +1,4 @@
+# Format:
+# module:released-version:current-version
+
+cloud-sql-jdbc-socket-factory:1.13.1:1.13.2-SNAPSHOT
```

Comment:
This looks like we need to add all our modules here.

Reply:
Done.